doc/source/tutorial/interpolate/1D.rst

```diff
@@ -339,6 +341,52 @@ same data as in the previous example:
 ..             `next`.
 
 
+.. _tutorial-interpolate_interp1d_replacements:
+
+Recommended replacements for `interp1d` modes
+---------------------------------------------
+
+As mentioned, `interp1d` class is *legacy*: we have no plans to remove it; we
+are going to keep supporting its existing usages; however
+we believe there are better alternatives which we recommend using in new code.
+
+Here we list specific recommendations, depending on the interpolation ``kind``.
+
+**Linear interpolation, ``kind="linear"``**
```

Comment:
Hmm it also looks like monospace doesn't render correctly within bold
![1000008113](https://github.com/user-attachments/assets/d8e2fe27-5b2d-451b-928d-1db34285fc2d)

Reply:
Grrr, third time the charm, hopefully! The current version (again, fixed and force-pushed) renders OK locally, let's see if CircleCI concurs. Sorry for the back-and-forth.

Reply:
> let's see if CircleCI concurs

... and it does.